src/hooks/use-user-session.ts

```diff
@@ -1,37 +1,9 @@
-import {useAtom} from 'jotai';
-import {UserSession, UserData, FinishedAuthData, showConnect} from '@stacks/connect-react';
-import {userSessionAtom, userDataAtom} from '../store';
+import {useAuth} from '@micro-stacks/react';
 
-import {appConfig, baseAuthOptions} from '../constants';
-
-const useUserSession = (): [UserSession | null, UserData | null, () => void, () => void] => {
-    const [userSession, setUserSession] = useAtom(userSessionAtom);
-    const [userData] = useAtom(userDataAtom);
-
-    const onFinish = async (payload: FinishedAuthData) => {
-        setUserSession(payload.userSession);
-    };
-
-    const openAuth = () => {
-        const authOptions = {
-            onFinish,
-            userSession: new UserSession({appConfig}),
-            ...baseAuthOptions
-        };
-        setUserSession(null);
-        showConnect(authOptions);
-    };
-
-    const signOut = () => {
-        if (!userSession) {
-            return;
-        }
-
-        setUserSession(null);
-        userSession.signUserOut();
-    }
-
-    return [userSession, userData, openAuth, signOut];
+// just for migration
+const useUserSession = (): [null, boolean, () => void, () => void] => {
+    const {openAuthRequest, isSignedIn, signOut} = useAuth()
```

Comment:
@aulneau can we keep UserSession and UserData here? We'll need them in the near future for gaia storage integration.

Reply:
@talhasch we don't need userSession or userData for gaia with micro-stacks, there are hooks for that (`usePutFile`, `useGetFile`, and soon some more specific `useStorage` helpers).

Can you tell me more about how you want to use gaia? I'm working on some new tooling right now and having some real use cases in mind would be super helpful

Reply:
This is early integration of gaia we disabled during development since it was slowing down progress: https://github.com/Trust-Machines/multisafe-ui/blob/main/src/hooks/use-storage.ts#L25